src/components/FloatingHelper/content/index.ts

```diff
@@ -0,0 +1 @@
+export * from './HelperContent';
```

Comment:
The files hierarchy is a bit strange. we have a component inside `FloatingHelper/content/`.  If you prefer, maybe it can `FloatingHelper/components/` in order to emphasize that this is an internal component

Reply:
I can rename `content` to `HelperContent` which is the name of the component. Which is like we do it in WSR (for example with RadioGroup and RadioButton)